packages/daimo-api/src/server/pushNotifier.ts

```diff
@@ -217,21 +243,30 @@ export class PushNotifier {
     }
     const otherStr = getAccountName(otherAcc);
 
-    const title =
-      value < 0
-        ? `Sent $${dollars} to ${otherStr}`
-        : `Received $${dollars} from ${otherStr}`;
-    let body;
-
-    if (memo) {
-      body = memo;
-    } else if (receivingRequestedMoney) {
-      body = `Your ${dollars} ${tokenSymbol} request was fulfilled`;
-    } else if (value < 0) {
-      body = `You sent ${dollars} ${tokenSymbol} to ${otherStr}`;
-    } else {
-      body = `You received ${dollars} ${tokenSymbol} from ${otherStr}`;
-    }
+    const title = (() => {
+      if (preSwapTransfer) {
+        assert(amount > 0); // foreignCoin can only be involved in receiving ends of swaps
+        return `Accepted $${dollars} from ${otherStr}`;
```

Comment:
so an example push notification is

> **Accepted $42.00 from 0x...**
> You accepted 10000 DEGEN as $42.00 USDC

?

Reply:
Yep, see the new tests for the copy:

<img width="740" alt="image" src="https://github.com/daimo-eth/daimo/assets/6984346/7ec3e936-afda-4e4e-8b89-1daa82400e29">